PublicMapping/districtbuilder
# Add toast library for notifications

## Overview

Add toast library for notifications. This addresses a need to display errors on the map page, and will be a generally useful way to communicate with the user on the map page.

### Demo

![image](https://user-images.githubusercontent.com/1809908/90005536-5ab76f00-dc65-11ea-89fc-774163dc262d.png)

### Notes

This uses the library `react-toastify`. Check out their [documentation](https://fkhadra.github.io/react-toastify/introduction) for more information.

I created a demo page at `/toast-demo` as a temporary place to show how this component works. Once it's implemented in the application, we can remove that demo page. I'd also be happy to delete the demo before I merge this PR, if you'd prefer.

## Testing Instructions

- Open application
- Navigate to `/toast-demo`
- Click on the buttons to see the toast notifications

Closes #188
